AdobeStockImageAdminUi/Test/Mftf/Test/AdminSeeMoreFromSeriesTest.xml

```diff
@@ -0,0 +1,41 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!--
+ /**
+  * Copyright © Magento, Inc. All rights reserved.
+  * See COPYING.txt for license details.
+  */
+-->
+
+<tests xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+       xsi:noNamespaceSchemaLocation="urn:magento:mftf:Test/etc/testSchema.xsd">
+    <test name="AdminSeeMoreFromSeriesTest">
+        <annotations>
+            <features value="AdobeStockImagePanel"/>
+            <useCaseId value="https://github.com/magento/adobe-stock-integration/issues/1006"/>
+            <title value="Cover User clicks See more button on Preview of the same image for the second time"/>
+            <description value="https://studio.cucumber.io/projects/131313/test-plan/folders/943908/scenarios/4314856"/>
```

Comment:
Please add the `testCaseId` and provide verbal information on the test in the desctiption